ADotNet.Infrastructure.Build/Program.cs

```diff
@@ -26,6 +27,7 @@ static void Main(string[] args)
                 {
                     Push = new PushEvent
                     {
+                        Tags = new string[] { "RELEASE" },
```

Comment:
We will need to change this to 

```suggestion
                        Tags = new string[] { "", "*", "RELEASE" },
```

With this configuration, the workflow will be triggered only when a push event occurs on the main branch with either no tag, a tag with any name (*), or a tag with the name RELEASE.

Reply:
We will need to change this to 

```suggestion
                        Tags = new string[] { "", "*", "RELEASE" },
```

With this configuration, the workflow will be triggered only when a push event occurs on the main branch with either no tag, a tag with any name (*), or a tag with the name RELEASE.

Reply:
I have checked this and I don't think we need to add a '*', because the workflow runs when a push event is triggered for a release tag OR on the main branch. They don't do AND between branches and tags.